SampleProject/VOKCoreDataManagerTests/ManagedObjectImportExportTests.m

```diff
@@ -674,4 +722,13 @@ - (NSArray *)customMapsArrayWithKeyPaths
              [VOKManagedObjectMap mapWithForeignKeyPath:COOL_RANCH_KEYPATH_KEY coreDataKey:COOL_RANCH_DEFAULT_KEY]];
 }
 
+- (NSArray *)customMapsArrayWithoutMicroseconds
+{
+    return @[[VOKManagedObjectMap mapWithForeignKeyPath:FIRST_NAME_DEFAULT_KEY coreDataKey:FIRST_NAME_DEFAULT_KEY],
```

Comment:
Consider using the mapping macro `VOK_MAP_FOREIGN_TO_LOCAL()`
defined here:
https://github.com/vokal/Vokoder/blob/master/Pod/Classes/VOKManagedObjectMap.h#L32